l10n_ar_txt_mendoza/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    'name': 'TXT Mendoza',
```

Comment:
I did not find anything more short :( 

rename to l10n_ar_txt_mendoza to (next options):

 a. l10n_ar_account_tax_settlement_mendoza
 b. l10n_ar_tax_settlement_mza
 b. l10n_ar_tax_mendoza

```suggestion
    'name': 'Tax settlement Mendoza',
```